src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int INT = -1;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)
Moreover, constants should have meaningful names